tests/check_odov3.sh

```diff
@@ -6,20 +6,20 @@ stackDirs=$(bash "$(pwd)/tests/get_stacks.sh")
 
 ginkgo run --procs 2 \
   --skip="stack: java-openliberty-gradle version: 0.4.0 starter: rest" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-cache-example-redhat" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-cache-example" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-circuit-breaker-example-redhat" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-circuit-breaker-example" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-crud-example-redhat" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-crud-example" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-configmap-example" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-http-example-redhat" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-secured-http-example" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-istio-circuit-breaker-booster" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-istio-distributed-tracing-booster" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-istio-routing-booster" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-istio-security-booster" \
-  --skip="stack: java-vertx version: 1.2.0 starter: vertx-messaging-work-queue-booster" \
```

Comment:
We'll need to keep both versions here, mainly so that this PR can pass since `1.2.0` was refactored in these changes.